keystores/src/main/java/com/palantir/conjure/java/config/ssl/DefaultCas.java

```diff
@@ -51,8 +56,12 @@
     private static Map<String, X509Certificate> getTrustedCertificates() {
         ImmutableMap.Builder<String, X509Certificate> certificateMap = ImmutableMap.builder();
         try {
+            InputStream certificateResource = DefaultCas.class.getResourceAsStream(CA_CERTIFICATES_CRT);
+            ByteArrayOutputStream inMemoryCertificates = new ByteArrayOutputStream();
+            ByteStreams.copy(certificateResource, inMemoryCertificates);
```

Comment:
ByteStreams.toByteArray avoids the ByteArrayOutputStream, alternatively guava Resources can give us `Resources.toByteArray(Resources.getResource(CA_CERTIFICATES_CRT))`.

I like the BufferedInputStream approach better because it avoids stream->bytea->bais, we simply wrap the original stream and don't need to worry about it.